ui/Screen/DesignSystemGuide.svelte

```diff
@@ -131,6 +131,24 @@
       value: 2,
     },
   ];
+
+  const infoNotification = params => {
+    return {
+      level: notification.Level.Info,
+      message: params.message,
+      showIcon: params.showIcon === undefined ? false : params.showIcon,
```

Comment:
```suggestion
      showIcon: params.showIcon || false,
```

I find this syntax more concise but it's up to you